client/blocks/site/index.jsx

```diff
@@ -146,18 +146,16 @@ class Site extends React.Component {
 						<div className="site__title">
 							{ /* eslint-disable wpcalypso/jsx-gridicon-size */ }
 							{ this.props.site.is_private && (
-								<span className="site__badge">
-									<Gridicon icon="lock" size={ 14 } />
-								</span>
+								<span className="site__badge site__badge-private">{ translate( 'Private' ) }</span>
 							) }
 							{ site.options && site.options.is_redirect && (
-								<span className="site__badge">
-									<Gridicon icon="block" size={ 14 } />
+								<span className="site__badge site__badge-redirect">
+									{ translate( 'Redirect' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Redirect', { context: 'post type singular name'} )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).